rollup.config.js

```diff
@@ -34,6 +34,7 @@ const createConfig = cliArgs => {
       sourcemap: true,
     },
     plugins: [
+      patchedFlowEntry({ mode: 'strict' }),
```

Comment:
The idea here is to include the flow types into the npm package.

<img width="661" alt="image" src="https://user-images.githubusercontent.com/1110551/57225266-2c7fc400-700c-11e9-8c37-48f017963456.png">

With `flow-copy-source`, we include the source files as `*.js.flow` files into the `dist/types` folder. The rollup plugin, simply creates a `*.js.flow` version of the bundle file that points to the `types` folder.
In theory, consumers of the package will be able to get the type annotations as flow understands the `*.js.flow` format and prefers to load that if it finds one.